hawkbit-rest-resource/src/main/java/org/eclipse/hawkbit/rest/resource/PagingUtility.java

```diff
@@ -69,6 +71,17 @@ static Sort sanitizeSoftwareModuleSortParam(final String sortParam) {
         return sorting;
     }
 
+    static Sort sanitizeSoftwareModuleTypeSortParam(final String sortParam) {
+        final Sort sorting;
+        if (sortParam != null) {
```

Comment:
if you early return you need no local variable and no else

   if (sortParam == null) {
         return new Sort(Direction.ASC, SoftwareModuleTypeFields.NAME.getFieldName());
      } 
        return  new Sort(SortUtility.parse(SoftwareModuleTypeFields.class, sortParam));